framework/wazuh/tests/data/schema_cve_test.sql

```diff
@@ -11,6 +11,8 @@ CREATE TABLE IF NOT EXISTS vuln_cves (
     version TEXT,
     architecture TEXT,
     cve TEXT,
+    type TEXT,
```

Comment:
Add new type and status values to schema below

Reply:
Arreglado